examples/oidc/roles/README.md

```diff
@@ -0,0 +1,38 @@
+# SSO Pathfinder Rolebased Client Using Node-Express
+
+## Description
+
+This is an example backend app that authenticates users using OAuth 2.0 standard flow, and helps the user to check their roles
+
+## Getting Started
+
+## Pre-requisites
+
+- You require an integration with client type `confidential` before you can start using this example app
+- Navigate to [SSO Onboarding](https://github.com/bcgov/sso-keycloak/wiki/SSO-Onboarding) to know more about creating an integration
+
+## Installing
+
+- Update the required values below and add them to a `.env` file in the project directory and save it .
+
+  ```sh
+  SSO_SESSION_SECRET=
+  SSO_AUTH_SERVER_URL=https://sso-keycloak-e4ca1d-dev.apps.gold.devops.gov.bc.ca/auth
```

Comment:
this URL is for sandbox that clients have no access. Please remove it. This information can be retrieved from the download JSON along with others. Thanks